README.md

```diff
@@ -116,6 +116,13 @@ If credentials are required, `buildctl` will attempt to read Docker configuratio
 buildctl build ... --exporter=local --exporter-opt output=path/to/output-dir
 ```
 
+#### Exporting OCI bundle to client
```

Comment:
At first glance I thought it creates a OCI Runtime Bundle (`rootfs` + `config.json`). :sweat_smile: 

So s/OCI bundle/tar archive with OCI Image Layout/ ?

Reply:
uh, s/tar achive/directory/

(Not in this PR, but probably we should allow client not to unpack the archive)